spec/features/idv/analytics_spec.rb

```diff
@@ -730,4 +841,61 @@ def wait_for_event(event, wait)
       end
     end
   end
+
+  context 'Happy selfie path' do
```

Comment:
This test, as it stands, can only test the desktop flow and cannot test SDK interaction. We don't seem to have a way to test / mock SDK interaction on the back end (and as I found, other "SDK-only" events like `Acuant SDK loaded` were not found in this file). This prevents us from testing the whole flow from front --> back. I plan to make a ticket for this.

Reply:
[Ticket for the above comment](https://cm-jira.usa.gov/browse/LG-11924).